deployment/Dockerfile.centos.all

```diff
@@ -0,0 +1,28 @@
+FROM centos:7
+# Docker build arguments
+ARG SOURCE_DIR=/jellyfin
+ARG ARTIFACT_DIR=/dist
+# Docker run environment
+ENV SOURCE_DIR=/jellyfin
+ENV ARTIFACT_DIR=/dist
+ENV DEB_BUILD_OPTIONS=noddebs
```

Comment:
```suggestion
```

Reply:
@JustAMan pointed this out too, but since this is a quick commit suggestion I'll apply it :)